anvil/src/input_handler.rs

```diff
@@ -586,6 +590,48 @@ impl AnvilState<UdevData> {
         }
     }
 
+    fn on_pointer_move_absolute<B: InputBackend>(
+        &mut self,
+        dh: &DisplayHandle,
+        evt: B::PointerMotionAbsoluteEvent,
+    ) {
+        let serial = SCOUNTER.next_serial();
+
+        let max_x = self
+            .space
+            .outputs()
+            .fold(0, |acc, o| acc + self.space.output_geometry(o).unwrap().size.w);
+
+        let max_h_output = self
+            .space
+            .outputs()
+            .max_by_key(|o| self.space.output_geometry(o).unwrap().size.h)
+            .unwrap();
+
+        let max_y = self.space.output_geometry(max_h_output).unwrap().size.h;
+
+        self.pointer_location.x = evt.x_transformed(max_x);
+        self.pointer_location.y = evt.y_transformed(max_y);
+
+        // clamp to screen limits
+        // this event is never generated by winit
```

Comment:
I don't get this comment, absolute move events are produced by winit as far as I remember, so I guess that this refers to the fact that winit does not require clamping?

Reply:
the comment was copied from code that already existed: https://github.com/Smithay/smithay/blob/670e8a3ab19103df0428db91d30f757e4262fa3c/anvil/src/input_handler.rs#L574

Since I'm calling the same function in new code which was called in the old code, I thought the comment remained relevant? I'm not really sure, I could remove this line https://github.com/Smithay/smithay/blob/670e8a3ab19103df0428db91d30f757e4262fa3c/anvil/src/input_handler.rs#L617 and leave the `clamp to screen limits` comment if that fits better?

Reply:
Yeah I think that would be better.

Given you can't push to this branch, I'll quickly do that.